qa/c8-orchestration-cluster-e2e-test-suite/utils/waitForItemInList.ts

```diff
@@ -15,22 +15,22 @@ export const waitForItemInList = async (
     shouldBeVisible?: boolean;
     timeout?: number;
     emptyStateLocator?: Locator;
-    clickAuthorizationsPageTab?: () => Promise<void>;
+    navigateAfterReload?: () => Promise<void>;
```

Comment:
There's nothing enforcing it to be a navigation on this callback
I would give a more generic name

```suggestion
    onAfterReload?: () => Promise<void>;
```